.github/workflows/fcos-build.yml

```diff
@@ -0,0 +1,34 @@
+name: FCOS Build from ${{ env.COPR_OWNER }}/${{ env.COPR_PROJECT }}
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  COPR_OWNER: "rhcontainerbot"
+  COPR_PROJECT: "podman-next"
+
+jobs:
+  wait-for-task:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up wait-for-copr
+      run: |
+        pip3 install git+https://github.com/packit/wait-for-copr.git@main
```

Comment:
I recommend against pinning this to `@main`.  This workflow is 99% dependent on this tool being available, installing correctly at runtime (risky in itself), and not having a recent bug.  Instead, I recommend pinning it to a tag, release version, or (abbreviated) commit hash.  Then I can help you (in another PR) teach renovate how to manage updates to the version.